multicluster/cmd/multicluster-controller/options.go

```diff
@@ -64,4 +96,5 @@ func (o *Options) setDefaults() {
 		HealthProbeBindAddress: ":8081",
 		LeaderElection:         false,
 	}
+	o.member = true
```

Comment:
why set member as true here? it will overwrite the setting from `MultiClusterConfig`

Reply:
I thought it sets only if it was not already set. So then o.options will also be overwritten? Maybe I should call it before reading config file